src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -436,6 +444,9 @@ export default {
 		},
 		addToBasket(payload) {
 			if (payload.success) this.$kvTrackEvent('loan-card', 'add-to-basket', 'filter-page-new-card');
+		},
+		handleTeamPicks(payload) {
+			this.showTeamPicks = payload;
```

Comment:
I don't think we want to set this to the payload here, since it's the toggle on whether to display the new icon.

Reply:
@roger-in-kiva still curious if this is necessary.

Reply:
I guess that removing recommended loans with the team picks switch, accordingly with comment below, we should remove the label... makes sense?

Reply:
Oh wait, is the payload here a boolean on whether to show the team picks?

Reply:
Oh yes, the default value is on if we show the challenge,
https://github.com/kiva/ui/blob/c80f11ea26927f992c1e0320fabb0cbf110b2e43/src/components/Lend/LoanSearch/TeamPicksSwitch.vue#L32

Reply:
Sounds good. I was thinking this was getting loan data or some other data based on interaction with the team picks.

Reply:
As not having BE well defined yet, difficult to do that. We can have a next iteration on this if needed.